app/models/tag.rb

```diff
@@ -75,6 +76,8 @@ class Tag < ActsAsTaggableOn::Tag
   #       this scope we have a name.
   scope :direct, -> { where(alias_for: [nil, ""]) }
 
+  scope :select_attributes_for_serialization, -> { select(ATTRIBUTES_FOR_SERIALIZATION) }
```

Comment:
thanks for adding this! looks better from the controller now that the Tag is responsible for working with its own constant